Makefile

```diff
@@ -32,6 +33,14 @@ vendor: $(JB_BIN) jsonnetfile.json jsonnetfile.lock.json
 	rm -rf vendor
 	$(JB_BIN) install
 
+crdschemas: vendor
+	./scripts/generate-schemas.sh	
+
+.PHONY: validate
+validate: crdschemas manifests $(KUBECONFORM_BIN)
+	# Follow-up on https://github.com/instrumenta/kubernetes-json-schema/issues/26 if validations start failing
+	$(KUBECONFORM_BIN) -schema-location 'https://kubernetesjsonschema.dev' -schema-location 'crdschemas/{{ .ResourceKind }}.json' -skip CustomResourceDefinition manifests/
```

Comment:
@paulfantom Just to confirm this is validating the CRDs, isn't it? Asking this because it came up as one of the requirements in the previous PR.

Reply:
yes, it is! :smile:  I am taking CRD definitions from libsonnet files in `vendor/` directory, manipulate them with `jq` to extract only `openAPIV3Schema` required by kubeconform and put those resulting files into `crdschema` directory from which `kubeconform` picks them up. All this is done in `scripts/generate-schema.sh` script.

Reply:
why do you have -skip CustomResourceDefinition here? Won't it.. skip the CRD? Could you add -summary to ensure the files are validate properly?